bluemira/equilibria/profiles.py

```diff
@@ -599,8 +599,10 @@ def __init__(
         R_0: float,
         B_0: float,
         shape: ShapeFunction | None = None,
+        use_approx_beta_p: bool = False,  # noqa: FBT001, FBT002
```

Comment:
So that a user has to type the variable name
```suggestion
        *,
        use_approx_beta_p: bool = False,  # noqa: FBT001, FBT002
```